tests/transactional/install_updates.pm

```diff
@@ -22,6 +22,7 @@ sub run {
 
     if (is_sle_micro) {
         assert_script_run 'curl -k https://ca.suse.de/certificates/ca/SUSE_Trust_Root.crt -o /etc/pki/trust/anchors/SUSE_Trust_Root.crt';
+        script_retry('pgrep update-ca-certificates', retry => 5, delay => 2, die => 0);
```

Comment:
I forgot sorry, you may need to include this function,
`use utils qw(script_retry);`

Reply:
Thanks! Good catch!